smart-connector/src/main/java/eu/knowledge/engine/smartconnector/impl/SmartConnectorImpl.java

```diff
@@ -21,6 +21,7 @@
 import eu.knowledge.engine.smartconnector.api.GraphPattern;
 import eu.knowledge.engine.smartconnector.api.KnowledgeBase;
 import eu.knowledge.engine.smartconnector.api.KnowledgeInteraction;
+import eu.knowledge.engine.smartconnector.api.MatchStrategy;
```

Comment:
Seems like this import can be removed as you don't reference it in the new code?